src/components/TimeoutButton.tsx

```diff
@@ -22,20 +22,26 @@ export const TimeoutButton: FC<Props> = ({
   timeoutSeconds = CONST.buttonTimeoutSeconds,
   onPress,
   containerStyle,
-  timeInBackground = 0,
 }) => {
   const [seconds, setSeconds] = useState(0);
+  const [timerOn, setTimerOn] = useState(false);
 
-  useEffect(() => {
-    setSeconds(seconds => seconds - timeInBackground);
-  }, [timeInBackground]);
+  const startTimer = () => {
+    BackgroundTimer.runBackgroundTimer(() => {
+      setSeconds(secs => {
+        if (secs > 0) return secs - 1;
+        else return 0;
+      });
+    }, 1000);
+  };
 
-  useInterval(
-    () => {
-      setSeconds(seconds - 1);
-    },
-    seconds > 0 ? 1000 : null,
-  );
+  useEffect(() => {
+    if (timerOn) startTimer();
```

Comment:
eslint is not complaning here about no brackets? personally I think brackets are more readable, we also use brackets in other places so it would be consistent to use them

Reply:
no eslint is fine but yes is done in way with brackets